src/main/java/Duke.java

```diff
@@ -1,10 +1,234 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+import Task.Task;
+import todo.todo;
+import Event.Event;
+import Deadline.Deadline;
+import Exception.*;
+
 public class Duke {
+
+    /*
+     * 
+     */
+    public static Task parseEcho(String echo) throws noTaskDescriptionException {
+        if (echo.startsWith("todo")) {
+            if (echo.substring(4).trim().isEmpty()) {
+                throw(new noTaskDescriptionException("     ☹ OOPS!!! The description of a todo cannot be empty."));
+            }
+            return new todo(echo.substring(4).trim());
+    
+        } else if (echo.startsWith("deadline")) {
+            String deadlineArguments = echo.substring(8).trim();
+            if (deadlineArguments.isEmpty()) {
+                throw(new noTaskDescriptionException("     ☹ OOPS!!! The description of a deadline cannot be empty."));
+            }
+            String splitArguments[] = deadlineArguments.split("/");
+            
+            return new Deadline(splitArguments[0], splitArguments[1].substring(2).trim());
+    
+        } else if (echo.startsWith("event")) {
+            String eventArguments = echo.substring(5).trim();
+            if (eventArguments.isEmpty()) {
+                throw(new noTaskDescriptionException("     ☹ OOPS!!! The description of a event cannot be empty."));
+            }
+            String splitArguments[] = eventArguments.split("/");
+            return new Event(splitArguments[0], splitArguments[1].substring(4).trim(), splitArguments[2].substring(2).trim());
+    
+        } else {
+            // System.out.println("Placeholder");
+            throw(new invalidInputException("      ☹ OOPS!!! I'm sorry, but I don't know what that means :-("));
+        }
+    }
+
+    /*
+     * 
+     */
+    static void printListNumber(ArrayList<Task> list) {
+        System.out.println("    Now you have " + list.size() + " task(s) in the list.");
+    }
+
+
+    public static Task parseFileReader(String echo, boolean isDone) {
+        if (echo.startsWith("todo")) {
+            return new todo(isDone, echo.substring(4).trim());
+        } else if (echo.startsWith("deadline")) {
+            String deadlineArguments = echo.substring(8).trim();
+            String splitArguments[] = deadlineArguments.split("/");
+            return new Deadline(isDone, splitArguments[0], splitArguments[1].substring(2).trim());
+        } else if (echo.startsWith("event")) {
+            String eventArguments = echo.substring(5).trim();
+            String splitArguments[] = eventArguments.split("/");
+            return new Event(isDone, splitArguments[0], splitArguments[1].substring(4).trim(), splitArguments[2].substring(2).trim());
```

Comment:
I think you can use line wrapping after the commas to improve readability (slightly above 120 char) 
```suggestion
            return new Event(isDone, splitArguments[0], 
                    splitArguments[1].substring(4).trim(), 
                    splitArguments[2].substring(2).trim());
```